github/azure-self-hosted-runners/README.md

```diff
@@ -0,0 +1,64 @@
+# Self-hosted Github Runners with Garm
+
+Those instructions deploy Garm for self-hosted github runners on azure (c)VMs. Garm listens to webhooks from github and spawns VMs on demand.
+
+## Build
+
+```bash
+docker build -t garm-azure .
+```
+
+## Service
+
+The application is deployed as an [ACI](https://azure.microsoft.com/en-us/products/container-instances) Container Group. It is bootstrapped with an init container which creates an admin user, an initial repository registration, and a pool for the repository. It keeps database state on a Storage Account-backed volume. The app will expose an HTTP/S ports for github webhooks.
+
+### Configuration
+
+The configuration is currently inlined in the `/init.sh` file. Not that you need to purge the state on the storage account on changes, otherwise initialization will be skipped.
+
+### Requirements
+
+- Terraform
+- Azure credentials, configured and logged in.
+- A github token scoped to `public_repo` to manage self-hosted runner registrations. 
+
+### Deployment
+
+Configure `tf/variables.tf` accordingly or set them in the env:
+
+```bash
+cd tf
+ export TF_VAR_github_token=mytoken
+ export TF_VAR_github_webhook_secret=mywebhooksecret
```

Comment:
We can ask user (in this case us devs) to create [terraform.tfvars](https://github.com/confidential-containers/cloud-api-adaptor/blob/staging/ci-infra/azure/terraform.tfvars) file. In the code we can provide `terraform.tfvars.example` ask them to copy it to `terraform.tfvars` and `.gitignore` this file so secrets are not accidentally checked-in.

Reply:
Where to get the webhook secret from? And it should be part of the `### Requirements` tab as well, isn't it?

Reply:
The webhook secret you define yourself and set it also in the repositories. Might need to add instructions